code/game/objects/items/devices/whistle.dm

```diff
@@ -28,14 +29,14 @@
 		..()
 
 /obj/item/device/whistle/proc/whistle_playsound(mob/user)
-	if (spamcheck)
+	if(!COOLDOWN_FINISHED(src, spam_cooldown))
+		to_chat(user, SPAN_DANGER("\ You are out of breath after using [src]! Wait [COOLDOWN_SECONDSLEFT(src, spam_cooldown)] second(s)."))
```

Comment:
```suggestion
		to_chat(user, SPAN_DANGER("You are out of breath after using [src]! Wait [COOLDOWN_SECONDSLEFT(src, spam_cooldown)] second\s."))
```

Reply:
Thanks Zonespace. I dont know how to code!